### DISCUSSION TITLE
Discussions are open!

### DISCUSSION THREAD

**Discussion by matmair** [inventree/InvenTree#2354] [General]
Hit there! This is a quick test of the existing discussion feature GitHub offers.
While issues and PRs are great, they are about code and discussions could be a better place for general discussions. We will see if that works out.

**Comment by matmair** [comment]
Testing tagging @SchrodingersGat @eeintech and my current main PR #2074 + its original issues #2037

**Comment by eeintech** [comment]
Fun! Thanks for getting this on :wink:
